packages/cli/src/commands/info.js

```diff
@@ -19,70 +21,112 @@ export default class InfoCommand extends Command {
     summary: 'list info',
     version: '1.0.0',
     args: [],
-    options: [
-      [ '--debug', 'include os versions in output' ],
-    ],
+    options: [['--debug', 'include os versions in output']],
   }
 
   constructor() {
     super(InfoCommand.definition)
   }
 
-  action(options, command) {
+  async action(options, command) {
     if (options.debug) {
-      console.debug('[CLI] Command \'%s\' called with options %o', this.name(), options)
+      console.debug(
+        '[CLI] Command \'%s\' called with options %o',
+        this.name(),
+        options
+      )
     } else {
       console.debug('[CLI] Command \'%s\' called', this.name())
     }
 
+    let versionFile = {}
+    try {
+      versionFile = JSON.parse(fs.readFileSync(VERSION_FILE))
+    } catch (error) {
+      console.warn(
+        `This project was generated with quire-cli version < 1.0.0.rc-8 so we cannot determine which cli or starter version was used to create it.`
```

Comment:
Perhaps the following
```suggestion
        `This project was generated with quire-cli prior to version 1.0.0.rc-8, the quire version file does not contain specific version information.`
```
In the case of a pre `rc-8` version file, or perhaps with a corrupt file, we might update the file with the new format?